content/blog/3-same-subject-continued-concerning-dangers/index.md

```diff
@@ -1,45 +1,40 @@
 ---
-title: The Same Subject Continued: Concerning Dangers from Foreign Force and Influence
+title: "The Same Subject Continued: Concerning Dangers from Foreign Force and Influence"
 date: "2020-11-18"
 originalAuthor: John Jay
 description: "It is better to have a single, united government when it comes to the dangers of war an international relations."
 ---
 
-It hardly takes a genius to recognize that the citizens of any country (especially intelligent and well-informed citizens like Americans) rarely adopt incorrect opinions about what is good for them--at least not for very long. So, the fact that our countrymen have long been united in believing that we should stay united under a single, sufficiently powerful federal government means that it's probably a good idea.
+It hardly takes a genius to recognize that the citizens of any country (especially a people as intelligent and well-informed as you, my fellow Americans) rarely adopt incorrect opinions about what is good for them--at least not for very long. So, the fact that we have long been united in believing that we should stay united under a single, sufficiently powerful federal government means that it's probably a good idea.
 
-Indeed, the more I look into the reasons for this opinion, the more convinced I am that they are compelling and decisive.
+In fact, the more I look into the reasons for this opinion, the more convinced I am that they are compelling and decisive.
 
-There are lots of different things that members of a democracy need to pay attention to, but providing for their collective safety is probably the first. "Safety" means lots of things to different people and in different situations, so it's important to define what I mean precisely.
+There are lots of different things that a democratic government needs to do, but providing for the safety of her citizens is the first. "Safety" means lots of things to different people and in different situations, so it's important to define what I mean precisely.
```

Comment:
How about "keeping its citizens safe" instead of "providing for the safety of its citizens"?